cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/ExecutorConfig.java

```diff
@@ -524,6 +524,15 @@ public final class ExecutorConfig {
       + "per broker concurrency cap on leadership movements will be divided by the concurrency adjuster (if enabled) when any "
       + "considered metric exceeds the concurrency adjuster limit.";
 
+  /**
+   * <code>max.num.brokers.above.metric.limit.no.decrease.cluster.concurrency</code>
+   */
+  public static final String MAX_NUM_BROKERS_ABOVE_METRIC_LIMIT_NO_DECREASE_CLUSTER_CONCURRENCY_CONFIG
+      = "max.num.brokers.above.metric.limit.no.decrease.cluster.concurrency";
```

Comment:
The name looks a bit weird to me, but I don't have a good idea on this config either. How about "min.num.brokers.violate.metric.limit.to.decrease.cluster.concurrency". Notice I intentionally change the name to "min"

Reply:
changed